segment/README.md

```diff
@@ -0,0 +1,60 @@
+# Segment
+
+This app sends an SMS to your number when your Segment-enabled website generates a specific tracking event.
```

Comment:
I'd add here or in the Prereqs that you'll need a Segment account and an event you want to track

Reply:
I added a Segment pre-requisites section to the README, which lists a Segment account and a Connection capable of handling `track` events.